accessors/spanner/mocks.go

```diff
@@ -38,6 +39,10 @@ type SpannerAccessorMock struct {
 	DropDatabaseMock                func(ctx context.Context, dbURI string) error
 	ValidateDMLMock                 func(ctx context.Context, query string) (bool, error)
 	TableExistsMock                 func(ctx context.Context, tableName string) (bool, error)
+	GetDatabaseNameMock             func() string
+	RefreshMock                     func(ctx context.Context, dbURI string)
+	SetSpannerClientMock            func(spannerClient spannerclient.SpannerClient)
```

Comment:
This seems weird. You shouldn't be needing to do this. Are you not able to set the `spannerClient` as part of the constructing the `spannerAccessor`? 

I think we should be constructing immutable instances of `spannerAccessor`. If you need a different spannerClient for any reason, create a different spannerAccessor.

Reply:
This is required by ExpressionVerificationAccessorImpl. Without this, we will not be able to use the interface within ExpressionVerificationAccessorImpl.


https://screenshot.googleplex.com/7THJ3K5n45d7fn5